com.unity.visualeffectgraph/Editor/SamplesLinkPackageManagerExtension.cs.meta

```diff
@@ -0,0 +1,11 @@
+fileFormatVersion: 2
+guid: 4ae080a8a6a24a9418d6cfc4f4ad403f
```

Comment:
this guid is the same as the guid for the meta file in the terrain tools package. it needs to be updated and pushed, one should be generated by default